app/enterprise/1.5.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -0,0 +1,179 @@
+restarts. Because etcd provides this functionality in Kong for Kubernetes
+deployments, it is not necessary to run an additional database, reducing
+maintenance and infrastructure requirements.
+
+While Kong for Kubernetes does not require a database, it is fully compatible
+with PostgreSQL and requires it for some features. etcd remains the source of
+truth, but the controller can translate Kubernetes resources from etcd into either
+DB-less or database configuration.
+
+## Choosing between DB-less or database-backed deployments
+
+In general, DB-less deployments are simpler to maintain and require less
+resources to run, and as such are the preferred option for Kong for Kubernetes.
+These deployments use the `kong-enterprise-k8s` image and must set
+`KONG_DATABASE=off` in their environment variables.
+
+Database-backed deployments offer a wider range of features using the
+`kong-enterprise-edition` image. Review the sections below to determine if your
+use case requires a feature that is not available in DB-less deployments.
+
+### Feature availability
+
+Some Kong Enterprise features are not available in DB-less deployments.
+Use the `kong-enterprise-edition` image and a database-backed deployment
+if you want to use:
+
+* Kong Manager
+* Dev Portal
+* Brain
+* Immunity
+* Teams (RBAC)
+* Vitals
+* Workspaces
+
+Because Kong for Kubernetes is configured by the ingress controller, some
+functionality in these features is different from traditional deployments:
+
+* Users will not typically create proxy configuration through Kong Manager;
+  proxy configuration is normally managed by the controller and users provide
+  configuration via Kubernetes resources.
+* Because the controller creates proxy configuration on behalf of users, users
+  do not need to interact with the Admin API directly. Typical Kong for
+  Kubernetes deployments do not expose the Admin API to protect it in lieu of
+  RBAC: only the controller can access it. Kubernetes-level RBAC rules and
+  namespaces should be used to restrict what configuration administrators can
+  create.
+* Ingress controller instances create configuration in a single workspace only
+  (`default` by default). To use multiple workspaces, deploy
+  multiple controller instances, setting the `CONTROLLER_KONG_WORKSPACE`
+  environment variable to the workspace that instance should use. These
+  instances should set `CONTROLLER_INGRESS_CLASS` to unique values, to avoid
+  creating duplicate configuration in workspaces. Note that if controller
+  instances are deployed outside the Kong pod the Admin API must be exposed,
+  and users should enable RBAC with workspace admin users for the controllers.
+  Set `CONTROLLER_KONG_ADMIN_TOKEN` to the RBAC user's token.
+* The controller cannot manage configuration for the features above: it cannot
+  create workspaces, Dev Portal content, admins, etc. These features must be
+  configured manually through the Admin API.
+
+### Plugin compatibility
+
```

Comment:
Let's point to the new document here to avoid duplication.

Reply:
https://github.com/Kong/kubernetes-ingress-controller/blob/master/docs/references/plugin-compatibility.md